client/my-sites/customer-home/controller.jsx

```diff
@@ -45,7 +45,10 @@ export async function maybeRedirect( context, next ) {
 	// disabled. Because of this, we refetch site information and limit traffic by scoping down
 	// requests to launchpad enabled sites.
 	// See https://cylonp2.wordpress.com/2022/09/19/question-about-infinite-redirect/#comment-1731
-	if ( maybeStalelaunchpadScreenOption && maybeStalelaunchpadScreenOption === 'full' ) {
+	if (
+		( maybeStalelaunchpadScreenOption && maybeStalelaunchpadScreenOption === 'full' ) ||
+		( maybeStalelaunchpadScreenOption && maybeStalelaunchpadScreenOption === 'new' )
```

Comment:
it seems like we would be checking 'new' in the same places we check 'full', im unsure why we need the 'new' variation

Reply:
`New` is just acting as a flag to indicate that a site is going through launchpad onboarding but has yet to be toggled to `full` or `off`.

It would represent the first time myHome is being loaded for the Launchpad site, similar to what you're describing here. p1674678377257579/1674672214.390509-slack-CHN6J22MP

Ideally we wouldn't co-opt the `launchpad_screen` option for this flag. We'd probably want to create a new settings option called something like `isSettingUpLaunchpad`.